service-base/src/main/java/org/eclipse/hono/service/amqp/AmqpEndpoint.java

```diff
@@ -42,4 +42,11 @@
      * @param sourceAddress The (remote) source address from the client's AMQP <em>ATTACH</em> message.
      */
     void onLinkAttach(ProtonConnection connection, ProtonSender sender, ResourceIdentifier sourceAddress);
+
+    /**
+     * Handles a lost connection.
+     * 
+     * @param connection The connection which is lost.
+     */
+    void onConnectionLost(ProtonConnection connection);
```

Comment:
Can we rename this method to something more specific to the circumstances under which it will be called, e.g. `onClientDisconnect`?

Reply:
I did change the name to `onConnectionClosed`, which aligns with the rest of the methods named like `publishConnectionClosedEvent`, which effectively calls this method.

`onClientDisconnect` sounds like some active action of a requested disconnection. Which may or may not be the case. "connection closed" or "connection lost" indicates a more passive scenario to me. Of course "lost" would most likely imply an unexpected loss of connection. Where "closed" is more neutral.